scripts/build_both_dylib_and_so.sh

```diff
@@ -11,14 +11,14 @@ SERVER=gsa6.broadinstitute.org
 LIB_PATH="src/main/c"
 
 echo "building local files"
-./gradlew clean build
+./gradlew clean build || exit 1
```

Comment:
Oups!, totally unaware of the "set -e" magic... I think at some point it was progressing farther than the first error but perhaps is just I wasn't interpreting the error/output correctly, happy to revert it.